packages/toolkit/src/lib/react-query-service/knowledge/useCreateKnowledgeBase.ts

```diff
@@ -0,0 +1,32 @@
+import { useMutation } from "@tanstack/react-query";
+import { createInstillAxiosClient } from "../../vdp-sdk/helper";
+import { KnowledgeBase } from "../../vdp-sdk/knowledge/types";
+
+async function createKnowledgeBaseMutation({
+  payload,
+  ownerId,
+  accessToken,
+}: {
+  payload: {
+    name: string;
+    description: string;
+    tags?: string[];
+  };
+  ownerId: string;
+  accessToken: string | null;
+}): Promise<KnowledgeBase> {
+  if (!accessToken) {
+    return Promise.reject(new Error("accessToken not provided"));
+  }
+  const client = createInstillAxiosClient(accessToken, true);
+  const response = await client.post<{
+    knowledge_base: KnowledgeBase;
+  }>(`/namespaces/${ownerId}/knowledge-bases`, payload);
+  return response.data.knowledge_base;
+}
```

Comment:
@thewbuk We usually maintain the query separately in the lib/vdp-sdk and now in packages/sdk

Please mimic the style in the sdk packages 

You can have a arifact folder under the packages/sdk/src and put all the query in the ArtifactClient. 

In this way, our user can also have a sdk about artifact to use, if they need it 

PS: All the react-query-services have this issue